browser/extensions/api/brave_extensions_api_client.cc

```diff
@@ -5,15 +5,25 @@
 
 #include "brave/browser/extensions/api/brave_extensions_api_client.h"
 
+#include "base/strings/string_piece.h"
 #include "extensions/common/permissions/permissions_data.h"
+#include "url/origin.h"
```

Comment:
It looks like you need to add a dep for `//url` in `browser/extensions/BUILD.gn`

Reply:
Added in last push.